dataeng/resources/pipeline-acceptance-test-manual.sh

```diff
@@ -39,8 +39,8 @@ env | sort
 
 mkdir -p $VENV_ROOT
 
-[ -f $TASKS_BIN/activate ] || virtualenv -p /usr/local/lib/python2.7.12/bin/python $VENV_ROOT/analytics-tasks
-virtualenv -p /usr/local/lib/python2.7.12/bin/python $VENV_ROOT/analytics-exporter
+[ -f $TASKS_BIN/activate ] || virtualenv -p python2.7 $VENV_ROOT/analytics-tasks
+virtualenv -p python2.7 $VENV_ROOT/analytics-exporter
```

Comment:
was this change intentional?  Looks good to me, but just wasn't in the commit description so I wasn't sure.  I assume something you did made python 2.7 appear in the normal PATH?

Reply:
Yes, it was intentional. Let me amend the commit message.